webhook/README.md

```diff
@@ -0,0 +1,21 @@
+# Prismic Webhook
+
+This is a Lambda which expects to live at a [Function URL](https://docs.aws.amazon.com/lambda/latest/dg/lambda-urls.html) which is configured as a webhook in Prismic.
+
+It receives Webhook events, checks that they are valid and contain a specified secret, and then forwards them to an EventBridge bus.
```

Comment:
Maybe worth a sentence explaining what the secret is for? I remember being confused by it when you mentioned it.

Reply:
And toss in a link to the Prismic docs for their webhooks: https://prismic.io/docs/webhooks